index.d.ts

```diff
@@ -95,7 +95,7 @@ export interface Space {
 export interface SyncCollection {
     entries: Array<Entry<any>>;
     assets: Array<Asset>;
-    deletedEntries: Array<Entry<any>>;
+    deletedEntries: Array<any>;
     deletedAssets: Array<Asset>;
```

Comment:
The same should be for `deletedAssets` - they won't resolve with the whole Asset, but only a meta-description